apps/src/lib/ui/ConfirmEnableMakerDialog.jsx

```diff
@@ -29,9 +43,15 @@ export class ConfirmEnableMakerDialog extends Component {
     return (
       <Dialog
         isOpen={this.props.isOpen}
-        confirmText={msg.enable()}
-        onConfirm={this.props.handleConfirm}
-        onCancel={this.props.handleCancel}
+        confirmText={experiments.isEnabled('microbit') ? null : msg.enable()}
```

Comment:
we should cache the value of `experiments.isEnabled('microbit')` -- we had a performance problem a while back because we were checking for an experiment on every render of the component. 

a couple options i can think of: move this calculation to `componentDidMount` and set the value in state, or move it to a parent component and pass it in as props